src/network/taraxa_capability.cpp

```diff
@@ -1690,39 +1700,44 @@ void TaraxaCapability::requestPbftNextVotes(NodeID const &peerID, uint64_t const
   sealAndSend(peerID, GetPbftNextVotes, RLPStream(2) << pbft_round << pbft_previous_round_next_votes_size);
 }
 
-void TaraxaCapability::sendPbftNextVotes(NodeID const &peerID) {
-  std::vector<Vote> next_votes_bundle = next_votes_mgr_->getNextVotes();
-  if (next_votes_bundle.empty()) {
-    LOG(log_er_next_votes_sync_) << "There are 0 next votes for previous PBFT round";
+void TaraxaCapability::sendPbftNextVotes(NodeID const &peerID, std::vector<Vote> const &send_next_votes_bundle) {
+  if (send_next_votes_bundle.empty()) {
     return;
   }
-  LOG(log_nf_next_votes_sync_) << "Send out size of " << next_votes_bundle.size() << " PBFT next votes to " << peerID;
 
-  RLPStream s(next_votes_bundle.size());
-  for (auto const &next_vote : next_votes_bundle) {
+  LOG(log_nf_next_votes_sync_) << "Send out size of " << send_next_votes_bundle.size() << " PBFT next votes to "
+                               << peerID;
+
+  RLPStream s(send_next_votes_bundle.size());
+  for (auto const &next_vote : send_next_votes_bundle) {
     s.appendRaw(next_vote.rlp());
     LOG(log_nf_next_votes_sync_) << "Send out next vote " << next_vote.getHash() << " to peer " << peerID;
```

Comment:
Lets change the log level here where we log each individual vote to DEBUG.

Reply:
Fixed